src/qibocal/cli/builders.py

```diff
@@ -58,9 +58,13 @@ def backend(self):
     def qubits(self):
         """Qubits dictionary."""
         if self.platform is not None:
-            return allocate_qubits(self.platform, self.runcard.qubits)
-
-        return self.runcard.qubits
+            if any(isinstance(i, list) for i in self.runcard.qubits):
+                return allocate_qubits_pairs(self.platform, self.runcard.qubits)
+            else:
+                return allocate_single_qubits(self.platform, self.runcard.qubits)
+        # TODO: check if this is needed
+        else:
+            return self.runcard.qubits
```

Comment:
```suggestion
            if any(isinstance(i, list) for i in self.runcard.qubits):
                return allocate_qubits_pairs(self.platform, self.runcard.qubits)
            return allocate_single_qubits(self.platform, self.runcard.qubits)
        # TODO: check if this is needed
        return self.runcard.qubits
```
These else were not needed. Also, what is this TODO?

Reply:
Indeed @rodolfocarobene the `else` can be removed.
The `TODO` is not necessary anymore. I believe that line is necessary when simulating the rb.